samples/container_customconfig_blockstorage/container-manifest.json

```diff
@@ -0,0 +1,203 @@
+                    "name": ""
+                },
+                {
+                    "matches": [
+                        {
+                            "type": "protocol",
+                            "value": "tcp"
+                        },
+                        {
+                            "type": "lport",
+                            "value": "5080"
+                        },
+                        {
+                            "type": "ip",
+                            "value": "0.0.0.0/0"
+                        }
+                    ],
+                    "actions": [
+                        {
+                            "drop": false,
+                            "limit": false,
+                            "limitValue": null,
+                            "limitrate": 0,
+                            "limitunit": "",
+                            "limitburst": 0,
+                            "portmap": true,
+                            "portmapto": {
+                                "appPort": 5080
+                            }
+                        }
+                    ],
+                    "name": ""
+                },
+                {
+                    "matches": [
+                        {
+                            "type": "protocol",
+                            "value": "tcp"
+                        },
+                        {
+                            "type": "lport",
+                            "value": "6080"
+                        },
+                        {
+                            "type": "ip",
+                            "value": "0.0.0.0/0"
+                        }
+                    ],
+                    "actions": [
+                        {
+                            "drop": false,
+                            "limit": false,
+                            "limitValue": null,
+                            "limitrate": 0,
+                            "limitunit": "",
+                            "limitburst": 0,
+                            "portmap": true,
+                            "portmapto": {
+                                "appPort": 6080
+                            }
+                        }
+                    ],
+                    "name": ""
+                }
+            ]
+        }
+    ],
+    "permissions": [],
+    "vmmode": "HV_PV",
+    "enablevnc": true,
+    "resources": [
+        {
+            "name": "resourceType",
+            "value": "Small"
+        },
+        {
+            "name": "cpus",
+            "value": "1"
+        },
+        {
+            "name": "memory",
+            "value": "1048576.00"
+        }
+    ],
+    "configuration": {
+        "customConfig": {
+            "name": "Custom_config",
+            "add": true,
+            "override": false,
+            "allowStorageResize": false,
+            "fieldDelimiter": "###",
+            "template": "SVA9IyMjSVAjIyM=",
```

Comment:
What is this? Is this a secret?

Reply:
Hi Kayan,

Thanks for pointing this out. I will remove this and make it as general.